src/Tools/ExternalAccess/Razor/Cohost/Handlers/DocumentHighlights.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Highlighting;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Cohost.Handlers;
+
+internal static class DocumentHighlights
+{
+    public static Task<DocumentHighlight[]?> GetHighlightsAsync(Document document, LinePosition linePosition, CancellationToken cancellationToken)
+    {
+        var globalOptions = document.Project.Solution.Services.ExportProvider.GetExports<IGlobalOptionService>().First().Value;
```

Comment:
Consider adding `GetFirstExport<T>()` and `GetExports<T>()` helper extension methods to clean up all these calls to `document.Project.Solution.Services.ExportProvider.GetExports<T>()`.

Reply:
Great idea. I called it `GetService` so it looks more normal :)

Reply:
That works, Although, I was actually thinking that the extension method would target `Document`, so you could write `document.GetExport<IGlobalOptionService>()` and save a lot of dotting. I'm not sure I agree that `GetService` looks all that normal when called off of an `ExportProvider`, where `GetExport` is the standard API.

Reply:
I'm happy to change it if thats the standard API, but in my mind GetExport would give me the Lazy, and GetService the actual impl.

Also, I know this is super constrained to Razor EA, but I think an extension on Document might be confusing because Roslyn has document services language services, workspace services etc. and this is specifically intended for broader MEF services, not Roslyn MEF. If that makes any sense.